packages/embed/src/css/includes/_close.scss

```diff
@@ -14,6 +14,8 @@
   color: #000;
   top: $top;
   right: $right;
+  background: none !important;
```

Comment:
What are this `!important` for?

Reply:
Good catch, I think they might not be necessary. In past customer CSS affected some elements in embeds, however this might not be the case.